geospaas_processing/downloaders.py

```diff
@@ -96,9 +96,8 @@ def check_and_download_url(cls, url, download_dir, **kwargs):
         if not file_name:
             raise DownloadError(f"Could not find file name for '{url}'")
         file_path = os.path.join(download_dir, file_name)
-        LOGGER.info("file_path %s", file_path)
+
         if os.path.exists(file_path) and os.path.isfile(file_path):
-            LOGGER.info("file exists")
```

Comment:
is that unnecessary that you remove it?

Reply:
that was a message I used for debugging when I was coding this, and I forgot to remove it